pkgs/top-level/all-packages.nix

```diff
@@ -18638,8 +18638,9 @@ in
 
   fingerd_bsd = callPackage ../servers/fingerd/bsd-fingerd { };
 
-  firebird = callPackage ../servers/firebird { icu = null; /*stdenv = gcc5Stdenv;*/ };
-  firebirdSuper = firebird.override { icu = icu58; superServer = true; /*stdenv = gcc5Stdenv;*/ };
+  inherit (callPackages ../servers/firebird { }) firebird_4 firebird_3 firebird_2_5;
+  firebird = firebird_3;
+  firebirdSuper = firebird.override { superServer = true; };
```

Comment:
Does this attribute even evaluate? The `override` is available for the value of `(callPackage /* ... */)`, but not `(callPackage /* ... */).foo`

Reply:
Thanks. I have extended the Super variant to all versions.